### DISCUSSION TITLE
How can we run the play application in debug mode?

### DISCUSSION THREAD

**Discussion by shubham54-sca** [Sunbird-Knowlg/Community#134] [Q&A]
URL - https://github.com/Sunbird-Knowlg/knowledge-platform/tree/master
We want to run the application in debug mode to get a better understanding of the code flow.

 cc: @AmiableAnil @Jayaprakash8887

**Comment by pallakartheekreddy** [comment]
@shubham54-sca Please use `mvn play2:run` command along with debug parameters and enable the debug configuration in your editor. The above command is giving error in some of the OS (Mac OSX). We are working on this issue. Will update here once the fix is released.
@maheshkumargangula

**Reply by pallakartheekreddy** [reply]
@shubham54-sca Hope your query is now resolved. Please mark the thread as answered.

**Reply by pallakartheekreddy** [reply]
@shubham54-sca As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
